src/httprpc.cpp

```diff
@@ -226,6 +241,8 @@ static bool InitRPCAuthentication()
     return true;
 }
 
+void JSONRPCRequestWalletResolver(JSONRPCRequest&, const HTTPRequest&);
```

Comment:
Looks after a strange design to forward decelerate this here, which is implemented in `rpcwallet.cpp` with not even a `#ifdef ENABLE_WALLET`